rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionCommand.java

```diff
@@ -4,10 +4,22 @@
 
 import java.util.Arrays;
 
+import rabbitescape.engine.err.RabbitEscapeException;
 import rabbitescape.engine.util.Util;
 
 public class SolutionCommand
 {
+    public final class WaitActionIllegalInMultiActionCommand extends RabbitEscapeException
+    {
```

Comment:
Should be called WaitActionInMultiActionCommand. The fact that it is illegal is not part of what went wrong.